benchto-driver/src/test/java/io/trino/benchto/driver/PrestoClientIntegrationTest.java

```diff
@@ -44,20 +46,21 @@ public void testPrestoClientLoadMetrics()
 
         List<Measurement> measurements = prestoClient.loadMetrics("test_query_id");
 
+        Map<String, String> attributes = Collections.singletonMap("scope", "prestoQuery");
         assertThat(measurements).containsExactly(
-                Measurement.measurement("prestoQuery-analysisTime", "MILLISECONDS", 21.07),
-                Measurement.measurement("prestoQuery-planningTime", "MILLISECONDS", 24.72),
-                Measurement.measurement("prestoQuery-totalScheduledTime", "MILLISECONDS", 66000.0),
-                Measurement.measurement("prestoQuery-totalCpuTime", "MILLISECONDS", 63600.0),
-                Measurement.measurement("prestoQuery-totalBlockedTime", "MILLISECONDS", 287400.0),
-                Measurement.measurement("prestoQuery-finishingTime", "MILLISECONDS", 69000.0),
-                Measurement.measurement("prestoQuery-rawInputDataSize", "BYTES", 1.34E9),
-                Measurement.measurement("prestoQuery-processedInputDataSize", "BYTES", 7.3961E8),
-                Measurement.measurement("prestoQuery-internalNetworkInputDataSize", "BYTES", 7.2961E8),
-                Measurement.measurement("prestoQuery-physicalInputDataSize", "BYTES", 1.35E9),
-                Measurement.measurement("prestoQuery-outputDataSize", "BYTES", 6900.0),
-                Measurement.measurement("prestoQuery-peakTotalMemoryReservation", "BYTES", 6800.0),
-                Measurement.measurement("prestoQuery-physicalWrittenDataSize", "BYTES", 462265065.0));
+                Measurement.measurement("analysisTime", "MILLISECONDS", 21.07, attributes),
```

Comment:
this will break scripts (jupyther). I think it's fine, but needs to be announced

Reply:
Not only scripts but our workflows as well.

Reply:
You mean alerts?